### PR TITLE
Typo + remove some warnings

### PR DESCRIPTION
**Pull request by lapineige**
- Wrong arm key.
- Yunohost minimum version is too low for packaging v2 + lots of helpers.

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
`chmod: cannot access '/var/www/garage/garage': No such file or directory`

The thing is this is not `install_dir` https://github.com/lapineige/garage_ynh/blob/patch-4/scripts/install#L72 but in /opt, which is `data_dir` I guess ?

File: manifest.toml [hunk 1]
```diff
@@ -59,7 +59,7 @@ ram.runtime = "50M"
     amd64.url = "https://garagehq.deuxfleurs.fr/_releases/v0.8.2/x86_64-unknown-linux-musl/garage"
     amd64.sha256 = "c15aa223b0fffb856c6e16276d1e77b0e94eb783a26eaeb7de80324f7c8b23c5"
     arm64.url = "https://garagehq.deuxfleurs.fr/_releases/v0.8.2/aarch64-unknown-linux-musl/garage"
-    amd64.sha256 = "9c419d47e87ac254a96cda6476e3c5c9102a9a2ea1ec1dc65d338bb46e415d78"
+    ard64.sha256 = "9c419d47e87ac254a96cda6476e3c5c9102a9a2ea1ec1dc65d338bb46e415d78"
```
Comment (alexAubin) [inline manifest.toml:62]:
```suggestion
    arm64.sha256 = "9c419d47e87ac254a96cda6476e3c5c9102a9a2ea1ec1dc65d338bb46e415d78"
```

**Comment by alexAubin** [comment]
Merging because this is straightforward

**Comment by lapineige** [comment]
> `chmod: cannot access '/var/www/garage/garage': No such file or directory`
> 
> The thing is this is not `install_dir` https://github.com/lapineige/garage_ynh/blob/patch-4/scripts/install#L72 but in /opt, which is `data_dir` I guess ?

Any idea about this ?

**Pull request merged**